src/main/java/com/openshift/jenkins/plugins/util/ClientCommandBuilder.java

```diff
@@ -91,12 +101,12 @@ private boolean hasArg(List<String> args, String... argsToFind) {
         // Some arguments may be long and provide little value (e.g. the path of
         // the server CA),
         // so hide them unless we are in logLevel mode.
-        if (!redacted || logLevel > 0) {
-            cmd.addAll(toStringArray(verboseOptions));
-            if (!hasArg(cmd, "--loglevel")) {
-                cmd.add("--loglevel=" + logLevel);
-            }
-        }
+        // if (!redacted || logLevel > 0) {
+        //     cmd.addAll(toStringArray(verboseOptions));
+        //     if (!hasArg(cmd, "--loglevel")) {
+        //         cmd.add("--loglevel=" + logLevel);
+        //     }
+        // }
```

Comment:
I could see swizzling this some because of what you did with `verboseOptions` (i.e. the cmd.addAll call), but completing removing the setting of `--loglevel` is not something we should remove, and I'm not seeing where it is set elsewhere here

And the use of asString(boolean) where the parameter is try still exists wrt the redacted var/param

Reply:
Good catch, forgot i left that commented out, fixed in next push